hack-sound-server/meson.build

```diff
@@ -0,0 +1,48 @@
+namespace = 'HackSoundServer'
+library_name = 'hacksoundserver'
+export_package_name = 'hack-sound-server'
+
+api_version = '0'
+api_name = '@0@-@0@'.format(library_name, api_version)
+libtool_version = '0.0.0'
+
+hacksoundserver_dbus_sources = gnome.gdbus_codegen(
+    export_package_name,
+    sources: 'data/com.endlessm.HackSoundServer.xml',
+    interface_prefix : 'com.endlessm.',
+    autocleanup: 'all',
+    install_header: true,
+    install_dir: join_paths(get_option('includedir'), meson.project_name())
+)
+
+hacksoundserverlib = shared_library(
+    api_name,
+    hacksoundserver_dbus_sources,
+    version: meson.project_version(),
+    dependencies: [gio_dep, gio_unix],
+    install: true,
+    soversion: api_version,
+    version: libtool_version
+)
+
+pkg.generate(
+    libraries: hacksoundserverlib,
+    version: meson.project_version(),
+    name: namespace,
+    filebase: namespace,
+    description: 'Clients for Hack DBus-based services',
+    requires: ['gio-2.0', 'gio-unix-2.0']
+)
+
+hack_services_gen_sources = gnome.generate_gir(
```

Comment:
This stanza is formatted differently than the rest of the file, it should be consistent.

Reply:
Still needs to be addressed.

Reply:
This is still not addressed, all the `generate_gir` stanzas still have different formatting than the rest of the file. Can you please make sure to fix up all comments from the previous review before asking for another review? (Or let me know why you decided not to fix them)